torch/testing/_internal/distributed/distributed_test.py

```diff
@@ -4936,6 +4936,52 @@ def _test_post_localSGD_optimizer_parity(self, create_averager, grad_is_view):
         def _create_periodic_model_averager(self):
             return averagers.PeriodicModelAverager(period=4, warmup_steps=10)
 
+        def _test_post_localSGD_optimizer_step_reload(self, create_averager):
+            learning_rate = 0.03
+            chkpt_file = os.path.join(os.environ["TEMP_DIR"], "checkpoint.pt")
```

Comment:
is the file cleaned up at end of unittest? If not, consider using python tempfiles: https://docs.python.org/3/library/tempfile.html